### ISSUE TITLE
disk_list input is very not intuitive to set up

### ISSUE THREAD

**Issue by DominiqueComte** [natankeddem/autopve#6]
Hi,

It took me some time to make `disk_list` work:

To get `disk_list` to correctly return `["sda", "sdb"]` in the answer, you have to input `[sda,sdb]` (without any quotes).
If you put any quotes in the web UI, they are escaped inside the answer: `["\"sda\"", "\"sdb\""]`.

Otherwise you get a `disk not found in disk_selection` error.

Once this was done, the installation worked very nicely, many thanks for this web UI.

**Comment by natankeddem** [comment]
Hello @DominiqueComte,
Thank you for the feedback. I will look into sanitizing the input.

**Comment by natankeddem** [comment]
@Konrni 
My mentality was to not use `"type"` as much as possible in the `set_key` method. By trying to use the auto detection in `set_key` I was hoping that future changes\additions that the PVE folks make will "just work". Right now the password field is the only key set to a fixed type and that is only to make sure that numeric only passwords are still treated as strings.

I just release a new version which should improve this situation. I had this sitting for a bit but I needed to spend some time playing with it to make sure I got the edge cases. The new input sanitization should remove single and double quotes as well as having better whitespace handling for list values. So users can use quotes or not; both should work now.
